java/src/jmri/jmrit/logixng/actions/swing/DigitalActionSwingBundle.properties

```diff
@@ -124,3 +124,15 @@ LogData_Format                      = Format
 LogData_TableAdd                    = Add
 LogData_ColumnType                  = Type
 LogData_ColumnData                  = Data
+
+ShowDialog_Buttons                  = Buttons
+ShowDialog_FormatType               = Format type
+ShowDialog_Format                   = Format
+ShowDialog_LocalVariable            = Local variable
+ShowDialog_Modal                    = Modal dialog
+ShowDialog_MultiLine                = Multi line
```

Comment:
either 'Multiline' or "Multi-line" is the standard usage, short for "Multiple Line" (two places)

Reply:
Fixed.